### ISSUE TITLE
Special character in the variable BRIDGE_USER_NAME

### ISSUE THREAD

**Issue by JoanoToxo** [nightscout/cgm-remote-monitor#8181]
Hello, I have an email as the value in the variable special character in the variable BRIDGE_USER_NAME. If I put an ampersand in BRIDGE_USER_NAME, bridge does not work, that is, it does not read the dexcom account data. If I put a BRIDGE_USER_NAME without the ampersand it works perfect

**Comment by oddst** [comment]
Dexcom has now changed their account registration process. It is no longer possible to register a username as earlier. Instead the username will automatically become (and cannot be changed) either your email address (with “@“ and “.”) or even worse your phone number in the format of “+4686751622”. How will BRIDGE or CONNECT support these changes?

**Comment by psonnera** [comment]
I though that was addressed some time ago... https://github.com/nightscout/nightscout.github.io/issues/79

**Comment by BrianWieder** [comment]
I don't think that usernames based off phone numbers currently work. My dexcom account username is my phone number starting with `+1`. Based on the startup logs of nightscount-connect, it removes the `+`.  I think it could be due to https://github.com/nightscout/cgm-remote-monitor/blob/21e0591d49235845acba58cf8b3cc7339921185b/lib/server/env.js#L210, since `!isNaN('+1') === true` therefore `value` is getting set to `Number('+1')` which is 1.

**Comment by CHR15714N** [comment]
Hasmthis issue been solved? I can't get it to work with my phone number.

**Comment by cameronm77** [comment]
Also wondering if there is a fix for the dexcom account with phone number  ?

**Comment by CHR15714N** [comment]
There is no fix, just a workaround.
All you can do is create a caregiver account in your account (a Sub-Account) and give it a nickname without symbols. Then login with the new account in your dexcom app and in nightscout.

**Comment by AndyLow91** [comment]
Thanks for raising this, and thanks to everyone who added detail around Dexcom accounts using email addresses or phone numbers as usernames.

I’m going to close this issue here because it is specifically about `BRIDGE_USER_NAME`, which belongs to the legacy `bridge` / `share2nightscout-bridge` integration. That path is deprecated in Nightscout, and new work should target the supported `connect` integration instead:

`CONNECT_SOURCE=dexcomshare`

There is clearly still a useful issue here around Dexcom account names containing special characters such as `+`, `@`, or phone-number-style usernames. Rather than fixing the deprecated bridge path, the right place to address that is in `nightscout-connect`, then update `cgm-remote-monitor` to consume the fixed `nightscout-connect` release.

For anyone still using the legacy bridge path today, the known workaround is to use a Dexcom caregiver/sub-account with a nickname that does not contain special characters. For the supported path, please use `CONNECT_SOURCE=dexcomshare`, and we can continue improving special-character login handling there.
